src/config/presets-loader.ts

```diff
@@ -0,0 +1,71 @@
+import path from 'path';
+import fse from 'fs-extra';
+
+import { Config } from './config-types';
+import { parseFile } from '@stackbit/utils';
+
+export class ConfigPresetsError extends Error {
+}
+
+export interface PresetsLoaderResult {
+    config: Config | null;
```

Comment:
The result probably can't have `null` for config because it already receives a valid Config.